demisto_sdk/commands/content_graph/interface/neo4j/queries/dependencies.py

```diff
@@ -56,6 +56,30 @@ def get_all_level_packs_dependencies(
     }
 
 
+def get_all_level_packs_imports(
```

Comment:
Instead of creating a new function that does (almost) the same thing, rename the current function for dependencies to `get_all_level_packs_relationships` which receives the same arguments + the relationship type, and call this function in both `all_level_depedencies` and `all_level_imports`, and delete the

Reply:
combined and rename get_all_level_relationships